aspnetcore/fundamentals/websockets.md

```diff
@@ -92,6 +92,32 @@ The WebSocket closed exception can also happen when returning too soon from an a
 
 Never use `Task.Wait`, `Task.Result`, or similar blocking calls to wait for the socket to complete, as that can cause serious threading issues. Always use `await`.
 
+::: moniker range=">= aspnetcore-6.0"
+
+### Using Compression
+
+> [!WARNING]
+> Enabling compression over encrypted connections makes the application subject to CRIME/BREACH type attacks.
```

Comment:
If it must be "CRIME/BREACH type attacks," then hyphenate it ("CRIME/BREACH-type"); otherwise ...

```suggestion
> Enabling compression over encrypted connections makes the app subject to CRIME/BREACH attacks.
```